.travis.yml

```diff
@@ -26,9 +24,8 @@ notifications:
     on_failure: always
 
 before_script:
-  - fluxbox >/dev/null 2>&1 &
   - pyenv global 3.7.1
-  - pip3 install --user localstack[full]
+  - pip3 install 'localstack==0.11.5' --force-reinstall
```

Comment:
actually we don't need this anymore right? and removing it will shave about a minute off build time

Reply:
urite